// models/Perfil.js
var mongoose = require('mongoose');
var Schema   = mongoose.Schema;
 
var PerfilSchema   = new Schema({
    username:   String,
    id:         String,
    hc:         Number,
    shc:        Number,
    pontuacao:  Number,
    touronegro: Number,
    koltsucuri: Number,
    botao1ok: Number,
    botao2ok: Number,
    botao3ok: Number,
    contadorarmas: Number,
    botao1arma1: Number,
	botao1arma2: Number,
	botao2arma1: Number,
    botao2arma2: Number,
    botao3arma1: Number,
	botao3arma2: Number,
	municaocloq17: Number,
	municaotouronegro: Number,
	municaokoltsucuri: Number,
	qtdeespecial: Number,
	qtdeenergetico: Number,
	qtdeqisuco: Number,
	sophiadesbloqueada: Number,
	sophiaselecionada: Number,
	royselecionado: Number,
	pontuacaopraia: Number,
	pontuacaoselva: Number,
	pontuacaoprefeitura: Number,
	pontuacaotemplo: Number,
	somaarmas: Number,
	somashc: Number,
	somahc: Number,
	sophia: Number,
	somaitem: Number,
	inimigosmortos: Number,
	okenergetico: Number,
	okespecial: Number,
	okqisuco: Number
	
	
	

  
});
 
module.exports = mongoose.model('Perfil', PerfilSchema);
